configs/usethis.json

```
@@ -1,31 +1,94 @@
 {
   "index_name": "usethis",
   "start_urls": [
-    "http://usethis.r-lib.org"
+    {
+      "url": "https://usethis.r-lib.org/index.html",
+      "selectors_key": "homepage",
+      "tags": [
+        "homepage"
+      ]
+    },
+    {
+      "url": "https://usethis.r-lib.org/reference",
+      "selectors_key": "reference",
+      "tags": [
+        "reference"
+      ]
+    },
+    {
+      "url": "https://usethis.r-lib.org/articles",
+      "selectors_key": "articles",
+      "tags": [
+        "articles"
+      ]
+    }
+  ],
+  "stop_urls": [
+    "/reference/$",
+    "/reference/index.html",
+    "/articles/$",
+    "/articles/index.html"
+  ],
+  "sitemap_urls": [
+    "https://usethis.r-lib.org/sitemap.xml"
   ],
-  "stop_urls": [],
   "selectors": {
-    "lvl0": {
-      "selector": ".contents h1",
-      "default_value": "Documentation"
+    "homepage": {
+      "lvl0": {
+        "selector": ".contents h1",
+        "default_value": "usethis Home page"
+      },
+      "lvl1": {
+        "selector": ".contents h2"
+      },
+      "lvl2": {
+        "selector": ".contents h3",
+        "default_value": "Context"
+      },
+      "lvl3": ".ref-arguments td, .ref-description",
+      "text": ".contents p, .contents li, .contents .pre"
+    },
+    "reference": {
+      "lvl0": {
+        "selector": ".contents h1"
+      },
+      "lvl1": {
+        "selector": ".contents .name",
+        "default_value": "Argument"
+      },
+      "lvl2": {
+        "selector": ".ref-arguments th",
+        "default_value": "Description"
+      },
+      "lvl3": ".ref-arguments td, .ref-description",
+      "text": ".contents p, .contents li"
     },
-    "lvl1": ".contents h2",
-    "lvl2": ".contents h3, .contents th",
-    "lvl3": ".contents h4",
-    "lvl4": ".contents h5",
-    "text": ".contents p, .contents li, .usage, .template-article .contents .pre"
+    "articles": {
+      "lvl0": {
+        "selector": ".contents h1"
+      },
+      "lvl1": {
+        "selector": ".contents .name"
+      },
+      "lvl2": {
+        "selector": ".contents h2, .contents h3",
+        "default_value": "Context"
+      },
+      "text": ".contents p, .contents li, .tempate-article .contents .pre"
+    }
   },
   "selectors_exclude": [
     ".dont-index"
-  ],
-  "sitemap_urls": [
-    "http://usethis.r-lib.orgsitemap.xml"
-  ],
+    ],
+  "min_indexed_level": 2,
   "custom_settings": {
-    "separatorsToIndex": "_"
-  },
-  "conversation_id": [
-    "641427970"
-  ],
-  "nb_hits": 1258
-}
\ No newline at end of file
+    "separatorsToIndex": "_",
+    "attributesToRetrieve": [
+      "hierarchy",
+      "content",
+      "anchor",
+      "url",
+      "url_without_anchor"
+    ]
+  }
```

```suggestion
   "conversation_id": [
      "641427970"
   ]
}
```
Please keep this parameter since we use it internally.